rundoctests.py

```diff
@@ -9,11 +9,49 @@
 import os
 import sys
 import doctest
-import resource
+from doctest import DocTestParser, OPTIONFLAGS_BY_NAME
+from multiprocessing import Process
+if os.name != 'nt':
+    import resource
 
 flags = doctest.ELLIPSIS
 
-filenames = sys.argv[1:]
+filenames = list(sys.argv[1:])
+if os.name == 'nt':
+    notinwindows = ['src/cysignals/pysignals.pyx', 'src/cysignals/alarm.pyx', 'src/cysignals/pselect.pyx']
+
+    for f in list(filenames):
+        if f in notinwindows:
+            filenames.remove(f)
+
+# Add an option to flag doctest what should not be run on windows.
+doctest.register_optionflag("SKIP_WINDOWS")
+doctest.register_optionflag("SKIP_POSIX")
+flag_skip_windows = OPTIONFLAGS_BY_NAME["SKIP_WINDOWS"]
+flag_skip_posix = OPTIONFLAGS_BY_NAME["SKIP_POSIX"]
+
+
+class WindowsSkipDocTestParser(DocTestParser):
```

Comment:
Why `Windows` in the name here? You handle `SKIP_WINDOWS` as well as `SKIP_POSIX`.

Reply:
You are right. I should have rename after adding `SKIP_POSIX`.